src/analytics.ts

```diff
@@ -342,43 +342,50 @@ export const logTiming = ({
 export const setScreenName = (
     screenName: string,
     params: {[key: string]: any} = {},
+    options?: LogEventOptions,
 ): Promise<void> => {
     if (!screenName) {
         console.warn('Missing analytics screenName');
         return Promise.resolve();
     }
 
+    const {sanitize = true} = {...defaultEventOptions, ...options};
```

Comment:
```suggestion
    const {sanitize} = {...defaultEventOptions, ...options};
```
it doesn't make sense to add a default value, because it's already defined in `defaultEventOptions`